core/src/main/java/org/frankframework/receivers/PullingListenerContainer.java

```diff
@@ -64,9 +63,10 @@ public class PullingListenerContainer<M> implements IThreadCountControllable {
 	private final AtomicInteger threadsRunning = new AtomicInteger();
 	private final AtomicInteger tasksStarted = new AtomicInteger();
 	private ResourceLimiter processToken = null; // guard against to many messages being processed at the same time
-	private Semaphore pollToken = null; // guard against to many threads polling at the same time
+	private ResourceLimiter pollToken = null; // guard against to many threads polling at the same time
 	private final AtomicBoolean idle = new AtomicBoolean(false); // true if the last messages received was null, will cause wait loop
 	private int retryInterval = 1;
+	private int maxThreadCount = 1;
```

Comment:
???

The ResourceLimiter I thought was meant as the unification of Semaphore + max nr of threads. So we need only one (and the nr of permits in the ResourceLimiter is always equal to the max nr of threads allowed).

This change is now reverting the concept we had to the dual counting mechanism that there was before.

Reply:
I was getting confused. I'll put it back, to keep the max allowed in the ResourceLimiter.